docs/multipart_reference.rst

```diff
@@ -7,7 +7,7 @@ Multipart reference
 
 .. class:: MultipartResponseWrapper(resp, stream)
 
-   Wrapper around the :class:`MultipartBodyReader` to take care about
+   Wrapper around the :class:`~aiohttp.MultipartBodyReader` to take care about
```

Comment:
Quick search shows that it's actually
```suggestion
   Wrapper around the :class:`MultipartReader` to take care about
```

Ref: https://github.com/aio-libs/aiohttp/pull/3905#discussion_r303313843.